Sources/NIO/ByteBuffer-core.swift

```diff
@@ -332,7 +332,7 @@ public struct ByteBuffer {
                 self._slice = _ByteBufferSlice(_slice.lowerBound ..< self._storage.capacity)
             }
         }
-        assert(self._slice.lowerBound + index + capacity <= self._slice.upperBound)
+        assert(self._slice.lowerBound + index + capacity <= self._slice.upperBound) // TODO(hh): fails on 32-bit
```

Comment:
Remove the TODO ?

Reply:
@helje5 why does this fail on 32-bit ? If that fails, the code above doesn't work

Reply:
Probably was a temporary note for myself and I fixed it on the calling side, need to check.